pyomo/gdp/disjunct.py

```diff
@@ -542,6 +543,17 @@ def __init__(self, *args, **kwargs):
                 "Cannot specify both rule= and expr= for Disjunction %s"
                 % ( self.name, ))
 
+    # FIXME: This is a workaround for double-declating the
+    # _algebraic_constraint autoslot for ScalarDisjunctions,
+    # necessitated by test failures encountered during release.
```

Comment:
Can we add the release number or year/month date to track the time frame of this comment?